README.md

```diff
@@ -1,52 +1,75 @@
-# CSI Snapshotter
+# CSI Snapshot Feature
 
-The CSI external-snapshotter is part of Kubernetes implementation of [Container Storage Interface (CSI)](https://github.com/container-storage-interface/spec).
+The CSI snapshot feature is part of Kubernetes implementation of [Container Storage Interface (CSI)](https://github.com/container-storage-interface/spec).
```

Comment:
I see a few main audiences for this readme:

1. Users wanting to use the snapshot feature.
2. CSI driver authors wanting to integrate snapshot support for their driver
3. Kubernetes distributions/deployers that want to support snapshots.
4. Developers making changes to the components.

For 1), I would point them to the K8s docs.

And then clearly state the audience for the rest of this README, which I think should be 2) and 3).

For 4), we should have another doc like "DEVELOPING.md"

Reply:
Sure.  I'll take a look of README of other sidecars.  I think the old README mirrored the one in external-attacher.